### PR TITLE
[Ready for review] Polish translation

### LINKED ISSUE #181: [Polish] Essential Components of Web Accessibility
I would like to  translate into [language]  the following resource:

- Page URL: [https://www.w3.org/WAI/fundamentals/components/](https://www.w3.org/WAI/fundamentals/components/)

I have read the information on Translating WAI Documents at https://www.w3.org/WAI/about/translating/

I will wait for confirmation that the resource is ready for translation.

### PR DESCRIPTION
**Pull request by zwiastunsw**
I added polish translated for: 
- Essential Components of Web Accessibility (_index.pl.md_)
- Descriptions of Essential Components of Web Accessibility Illustrations (_wai-components-example.pl.md_)

and
- graphics with Polish labels (svg version)

### CODE REVIEW

File: content/wai-components-example.pl.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
+---
+title: Opisy ilustracji w artykule Podstawoe komponenty dostępności Internetu
```
Comment (jcabak) [inline content/wai-components-example.pl.md]: Podstawowe

File: content/wai-components-example.pl.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## Opis ilustracji Wytyczne dla różnych komponentów {#guide}
+
+![Ilustracja przedstawia różne wytyczne dla różnych komponentów]({{ "/content-images/wai-components/specs-pl.svg" | relative_url }})
+
+Ilustracja przedstawia trzy graficznie oznaczone pola obrazujące treść i ludzi. Pośrodku na górze znajduje się pole oznaczone jako „treść” na którym są baletnica, zdjęcie dziecka, znacznik img, tekst, formularz, widżet pogody i wykres kołowy. Idąc od dołu z lewej strony, linia łączy  „tworców” z „treścią” na górze, przechodząc przez „narzędzia autorskie” i „narzędzia oceniające”. Idąc od dołu z prawej stronie, linia łączy „użytkowników” z „treścią” na górze, przechodząc przez „przeglądarki, odtwarzacze mediów” i „technologie wspomagające”.
```
Comment (jcabak) [inline content/wai-components-example.pl.md]: twórców, brak polskiego znaku

File: content/wai-components-example.pl.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Poniżej znajdują się „Wytyczne dla dostępności", które obejmują „ATAG” ze strzałką wskazującą na „narzędzia autorskie” i „narzędzia oceniające”, „WCAG” wskazujące na „treść” oraz „UAAG” wskazujące na „przeglądarki, odtwarzacze mediów” i „technologie wspomagające”. Na samym dole „Specyfikacje techniczne (HTML, ARIA, CSS, SVG, SMIL itp.)" tworzą podstawę ze strzałką wskazującą na wytyczne dla dostępności.
+
+{% comment %}
+@@  This is a redundant section. Repeated the relate.png image, but added a different description. (commented by zwiastunsw @@
```
Comment (jcabak) [inline content/wai-components-example.pl.md]: brak końca nawiasu

File: content/wai-components-example.pl.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+{% include_cached toc.html type="end" %}
+{:/}
+
+Na tej stronie opisano ilustracje w dokumencie “[[Podstawowe komponenty dostępności Internetu]](/standards/components/)” oraz prezentacji [Podstawowe komponenty dostępności Internetu](http://www.w3.org/WAI/intro/components-slides).
+
+## Opis ilustracji przedstawiającej powiązania między komponentami {#relate}
+
+![Ilustracje powiązania między komponentami]({{ "/content-images/wai-components/relate-pl.svg" | relative_url }})
+
+Ilustracja przedstawia trzy graficznie oznaczone pola obrazujące treść i ludzi. Pośrodku na górze znajduje się pole oznaczone jako „treść” na którym są baletnica, zdjęcie dziecka, znacznik img, tekst, formularz, widżet pogody i wykres kołowy. Idąc od dołu z lewej strony, linia łączy  „tworców” z „treścią” na górze, przechodząc przez „narzędzia autorskie” i „narzędzia oceniające”. Idąc od dołu z prawej stronie, linia łączy „użytkowników” z „treścią” na górze, przechodząc przez „przeglądarki, odtwarzacze mediów” i „technologie wspomagające”.
```
Comment (jcabak) [inline content/wai-components-example.pl.md]: twórców, brak polskiego znaku

**Review by jcabak — APPROVED** [review]

**Review by jcabak — APPROVED** [review]

**Comment by remibetin** [comment]
@zwiastunsw FYI, I have added the remaining instructions in https://github.com/w3c/wai-translations/issues/181

**Comment by zwiastunsw** [comment]
Related https://github.com/w3c/wai-website-data/pull/140

**Comment by zwiastunsw** [comment]
@remibetin 
The link to the illustration descriptions does not work for an incomprehensible reason.
An error is reported on line 4 during the build: 
``` 
YAML Exception reading /opt/build/repo/content/wai-components-example.pl.md: (<unknown>): 
did not find expected key while parsing a block mapping at line 4 column 1
```

**Comment by remibetin** [comment]
@zwiastunsw I have opened a PR to fix the problem, can you merge it? https://github.com/lepszyweb/wai-components/pull/1

Can you check "Allow edits by maintainers" so I can directly fix issues like this in your PRs?

File: content/index.pl.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+**Twórcy** stron internetowych zwykle używają **narzędzi autorskich** i narzędzi oceniających do tworzenia **treści** cyfrowych.
+
+**Ludzie** („**użytkownicy**”) używają **przeglądarek, odtwarzaczy mediów, technologii wspomagających** lub innych „**programów użytkownika**”, aby uzyskać dostęp do treści cyfrowych i&nbsp;wchodzić z&nbsp;nimi w&nbsp;interakcje.
+
+## Współzależności komponentów {#depend}
+
+Między tymi komponentami istnieją istotne współzależności; to znaczy że komponenty muszą ze sobą współpracować, aby Internet był dostępny. Na przykład, dla tekstu alternatywnego opisującego obraz:
+
+-   **specyfikacja techniczna** wymaga alternatywy tekstowej (np. HTML definiuje atrybut tekstu alternatywnego (`alt`) w elemencie obrazka (`img`)
+-   **wytyczne WAI** - WCAG, ATAG i UAAG, wymienione w dalszej części - określają, jak wdrożyć tekst alternatywny w&nbsp;różnych komponentach, aby zwiększyć dostępność
```
Comment (remibetin) [inline content/index.pl.md]:
```suggestion
-   **wytyczne WAI** - WCAG, ATAG i UAAG, wymienione w dalszej części - określają, jak wdrożyć tekst alternatywny w&nbsp;różnych komponentach, aby zwiększyć dostępność
```

A link is missing here. Original version is `([WCAG, ATAG, UAAG described below](#guidelines))`

File: content/index.pl.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+{% assign longdesc_url = "/fundamentals/components/examples/pl#weak" | relative_url %}
+{% include image.html src="bridge.pl.png" alt="Ilustruje, co dzieje się, gdy jeden komponent jest słaby" longdesc=longdesc_url %}
+
+Gdy jeden komponent kiepsko obsługuje dostępność, inne komponenty mogą to czasami zrekompensować poprzez „obejścia”, choć wymaga to znacznie większego wysiłku i ogólnie nie jest dobre dla dostępności. Na przykład:
+
+-   twórcy mogą wykonać więcej pracy, aby zrekompensować brak obsługi dostępności w&nbsp;narzędziach autorskich; na przykład poprzez bezpośrednie kodowanie znaczników zamiast za pomocą narzędzia,
+-   użytkownicy mogą wykonać więcej pracy, aby zrekompensować  brak obsługi dostępności w przeglądarkach, odtwarzaczach mediów lub technologiach wspomagających; na przykład, używając innej przeglądarki lub technologii wspomagającej, aby przezwyciężyć różne problemy z dostępnością.
```
Comment (remibetin) [inline content/index.pl.md:126]: @zwiastunsw A paragraph from the original version is missing in the translation:

> However, in most cases the works-arounds are not implemented and the result is still poor accessibility. Additionally, sometimes poor accessibility support in one component cannot be reasonably overcome by other components and the result is inaccessibility, making it impossible for some people with disabilities to use a particular website, page, or feature.

File: content/index.pl.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+
+## Wytyczne i inne standardy {#guidelines}
+
+Inicjatywa na Rzecz Dostępności Internetu [WAI](https://www.w3.org/WAI/)) w Konsorcjum <span lang="en">World Wide Web</span> ([W3C](https://www.w3.org/)) opracowuje **standardy dostępności cyfrowej** dla różnych komponentów:
```
Comment (remibetin) [inline content/index.pl.md]:
```suggestion
Inicjatywa na Rzecz Dostępności Internetu ([WAI](https://www.w3.org/WAI/)) w Konsorcjum <span lang="en">World Wide Web</span> ([W3C](https://www.w3.org/)) opracowuje **standardy dostępności cyfrowej** dla różnych komponentów:
```

A parenthesis is missing

File: content/index.pl.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+-   [[Wytyczne dla dostępności narzędzi autorskich (ATAG)]](/standards-guidelines/atag) dotyczące narzędzi do tworzenia treści
```
Comment (remibetin) [inline content/index.pl.md]:
```suggestion
-   [[Wytyczne dla dostępności narzędzi autorskich (ATAG)]](/standards-guidelines/atag/) dotyczące narzędzi do tworzenia treści
```

To fix a problem with the rendered link.

**Review by remibetin — CHANGES_REQUESTED** [review]
Thank you @zwiastunsw for this new translation!

I have added some comments for your consideration. 

Can you also please remove the `content.png`, `relate-developer.png`, `relate-example.png` & `relate-users.png` images from this PR, since they are not used in the resource?

Let me know when you have done the changes so I can check the PR again.

Thanks!

**Comment by zwiastunsw** [comment]
@remibetin thanks, fixed

**Review by remibetin — APPROVED** [review]

**Comment by remibetin** [comment]
This translation has been published today 🚀 Thank you @zwiastunsw!

See https://www.w3.org/WAI/fundamentals/components/pl & https://www.w3.org/WAI/fundamentals/components/examples/pl

**Comment by remibetin** [comment]
QA done: ✅

**Pull request merged**